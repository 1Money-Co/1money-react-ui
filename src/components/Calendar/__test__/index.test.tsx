import 'jsdom-global/register';
import * as React from 'react';
import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom';
import { Calendar } from '../index';

const originalConsoleError = console.error;
console.error = (message, ...optionalParams) => {
  if (
    message.includes('Could not parse CSS stylesheet') ||
    message.includes('findDOMNode is deprecated and will be removed')
  ) {
      return;
  }
  originalConsoleError(message, ...optionalParams);
};

describe('Calendar', () => {
  it('renders correctly', () => {
    const wrapper = render(
      <Calendar />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
