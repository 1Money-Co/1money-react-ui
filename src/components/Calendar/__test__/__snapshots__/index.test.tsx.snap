// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calendar renders correctly 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="usd1-react-ui-calendar-wrapper"
      >
        <span
          class="usd1-react-ui-calendar  false p-calendar p-component p-inputwrapper"
          data-pc-name="calendar"
          data-pc-section="root"
        >
          <input
            aria-autocomplete="none"
            aria-controls="pr_id_8_panel"
            aria-expanded="false"
            aria-haspopup="dialog"
            autocomplete="off"
            class="p-inputtext p-component"
            data-pc-name="inputtext"
            data-pc-section="root"
            inputmode="none"
            placeholder="MM/DD/YYYY"
            role="combobox"
            type="text"
          />
        </span>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="usd1-react-ui-calendar-wrapper"
    >
      <span
        class="usd1-react-ui-calendar  false p-calendar p-component p-inputwrapper"
        data-pc-name="calendar"
        data-pc-section="root"
      >
        <input
          aria-autocomplete="none"
          aria-controls="pr_id_8_panel"
          aria-expanded="false"
          aria-haspopup="dialog"
          autocomplete="off"
          class="p-inputtext p-component"
          data-pc-name="inputtext"
          data-pc-section="root"
          inputmode="none"
          placeholder="MM/DD/YYYY"
          role="combobox"
          type="text"
        />
      </span>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
